### ISSUE TITLE
Question about the generated data

### ISSUE THREAD

**Issue by QM20** [dkkmy/Midas#2]
Thanks for the excellent work. When reading your paper, I had a small question about the definition of "**_generated radar data_**". 

In the experiment, you mentioned the accuracy of the generated radar can achieve 84.67% with only the generated radar data. Is this generated radar data acquired after the transformer network? If that, how do you train such a transformer network without real-world paired radar data?

**Comment by dkkmy** [comment]
Thank you for your recognition of our work. The generated data can be used as training data to achieve an accuracy of 84.67%. At the same time, the correspondence refers to the data we actually collect, that is, the real radar data. In addition, there will be a camera to record real video during the collection process, and we will use the pipeline to generate radar data from the collected video to form a one-to-one correspondence with the real radar data to train the Transformer. Hope our answer can help you.

**Comment by QM20** [comment]
Thanks for the classification. I am wondering if you tested the accuracy of the coarse-generated radar data without the transformer refinement. That is, you can use the radar data calculated from the physical model and directly use them to train a classification model. Would the accuracy still be high in such a case?

**Comment by dkkmy** [comment]
Please refer to the paper for ablation experiments.
